src/app/docs/popover/popover-docs.component.html

```diff
@@ -0,0 +1,219 @@
+<sky-docs-demo-page
+  pageTitle="Popover"
+  moduleName="SkyPopoverModule"
+  packageName="@skyux/popovers"
+  gitRepoUrl="https://github.com/blackbaud/skyux-popovers"
+  sourceCodeLocation="src/app/public/modules/popover"
+>
+  <sky-docs-demo-page-summary>
+    The popover module displays small chunks of contextual content on pages or modals. It highlights content in an HTML-formatted popover that is similar to a tooltip but provides more room for plain text, HTML content, or Angular components. The popover trigger directive ties the popover to an element that users can interact with to display the popover.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-radio-group
+          heading="Placement"
+          propertyName="placement"
+          [initialValue]="'above'"
+          [choices]="placementChoices"
+        >
+        </sky-docs-demo-control-panel-radio-group>
+      </sky-docs-demo-control-panel-section>
+
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-radio-group *ngIf="showAlignmentOptions"
+          heading="Alignment"
+          propertyName="alignment"
+          [initialValue]="'center'"
+          [choices]="alignmentChoices"
+        >
+        </sky-docs-demo-control-panel-radio-group>
+
+        <sky-docs-demo-control-panel-checkbox
+          label="Show title"
+          propertyName="showTitle"
+          [checked]="true"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <button
+      class="sky-btn sky-margin-inline-default"
+      type="button"
+      [skyPopover]="myPopover0"
+      [skyPopoverAlignment]="demoSettings.skyPopoverAlignment"
+      [skyPopoverPlacement]="demoSettings.skyPopoverPlacement"
+    >
+      Popover demo on click
+    </button>
+
+    <button
+      class="sky-btn"
+      type="button"
+      [skyPopover]="myPopover0"
+      skyPopoverTrigger="mouseenter"
+      [skyPopoverAlignment]="demoSettings.skyPopoverAlignment"
+      [skyPopoverPlacement]="demoSettings.skyPopoverPlacement"
+    >
+      Popover demo on hover
```

Comment:
Is a button the right thing for the "hover" example?

Certainly, there's nothing wrong with applying a popover to a button - you can apply it to anything. However, my first instinct is to click the button, which hides the popover. Almost seems like something is wrong, even though there isn't.

Reply:
This was the design that Adam provided to me; happy to discuss different options if needed. One potential use case for this behavior is to provide the end-user with an explanation of the button before they click it, but I agree to some extent, especially since we have two buttons back-to-back that operate differently. But I suppose this is purely to demonstrate the behaviors.

Reply:
It's not a blocker by any means. Just thought ya'll should know my first reaction was it seemed like something was broken. @Blackbaud-AdamFunderburk any thoughts?

Reply:
Alex has a fair point. Changing that second button to use sky-btn-link might be all the contrast needed here, without overthinking it.

Reply:
Done!